src/main/java/edu/cornell/kfs/concur/ConcurParameterConstants.java

```diff
@@ -32,4 +32,7 @@
     public static final String CONCUR_REQUEST_EXTRACT_PDP_DOCUMENT_TYPE = "CONCUR_REQUEST_EXTRACT_PDP_DOCUMENT_TYPE";
     
     public static final String CONCUR_PROCESS_FAILED_EVENT_QUEUE = "CONCUR_PROCESS_FAILED_EVENT_QUEUE";
+    
+    public static final String CONCUR_REPORT_EMAIL_FROM_ADDRESS = "CONCUR_REPORT_EMAIL_FROM_ADDRESS";
+    public static final String CONCUR_REPORT_EMAIL_TO_ADDRESS = "CONCUR_REPORT_EMAIL_TO_ADDRESS";
```

Comment:
I suspect both sets of reports will be emailed to the same people, so I think we are OK.